### ISSUE TITLE
trezor: cmdline 'paytomany' doesn't work with trezor

### ISSUE THREAD

**Issue by molecular** [spesmilo/electrum#917]
issuing a paytomany command via the commandline interface results in this error:

```
Traceback (most recent call last):
  File "electrum", line 339, in <module>
    wallet = Wallet(storage)
  File "/mnt/maui/home/nick/bitcoin/electrum/lib/wallet.py", line 1620, in __new__
    raise BaseException('unknown wallet type', wallet_type)
BaseException: ('unknown wallet type', u'trezor')
```

**Comment by ecdsa** [comment]
this is because init_plugins is called only with the GUI

**Comment by ecdsa** [comment]
it works in the gui console now.  21ccb1e82de79ec5277e2f9ec55c30166d7c1475

**Comment by ELM4Ever** [comment]
Is it possible to have an update on this issue?

When running Electrum as a daemon, Electrum does not recognize Trezor as a valid wallet_type.

**Comment by ecdsa** [comment]
partially fixed by 8f98ea4acacfadb4d6188d503f5c8d9e6be86e1f

**Comment by ecdsa** [comment]
fixed in 2863d195794f87bf4496451829eb8b24ef19851a
